packages/firestore/test/integration/util/firebase_export.ts

```diff
@@ -121,5 +121,20 @@ const Timestamp = usesFunctionalApi()
   : legacyNamespace.Timestamp;
 const GeoPoint = usesFunctionalApi() ? exp.GeoPoint : legacyNamespace.GeoPoint;
 const Blob = usesFunctionalApi() ? exp.Blob : legacyNamespace.Blob;
+const loadBundle = usesFunctionalApi()
+  ? exp.loadBundle
+  : legacyNamespace.loadBundle;
+const namedQuery = usesFunctionalApi()
+  ? exp.namedQuery
+  : legacyNamespace.namedQuery;
 
-export { Firestore, FieldValue, FieldPath, Timestamp, Blob, GeoPoint };
+export {
+  Firestore,
+  FieldValue,
+  FieldPath,
+  Timestamp,
+  Blob,
+  GeoPoint,
+  loadBundle,
+  namedQuery
```

Comment:
You shouldn't need this if you update your integration tests to use `Firestore.loadBundle`. For this to work, you probably need to import `packages/firestore/index.bundle.ts` in this and the other firestore_export file.